### PR TITLE
Touch-ups to the Python installation guide

### PR DESCRIPTION
**Pull request by zanieb**

### CODE REVIEW

File: docs/guides/install-python.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Installing Python
 
-uv can manage Python installations — Python does not need to be installed to use uv.
+If Python is already installed, uv should [detect and use](#using-an-existing-python-installation) it without configuration. However, uv can also install and manage Python versions.
```
Comment (charliermarsh) [inline docs/guides/install-python.md:3]: "Install and manage Python for you", maybe?

File: docs/guides/install-python.md [hunk 2]
```diff
@@ -54,12 +60,12 @@ Even if a specific Python version is not requested, uv will download the latest
...
+<!-- TODO(zanieb): Restore when Python shim management is added
 Note that when an automatic Python installation occurs, the `python` command will not be added to the shell. Use `uv python install-shim` to ensure the `python` shim is installed.
+-->
 
 ## Using an existing Python installation
 
-uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior, uv will use the system Python if it satisfies the requirements of the command invocation.
-
-To force uv to use the system Python, provide the `--python-preference only-system` option.
+uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior, uv will use the system Python if it satisfies the requirements of the command invocation. See the [Python discovery](../python-versions.md#discovery-order) documentation for details.
```
Comment (charliermarsh) [inline docs/guides/install-python.md:69]: I think the comma after behavior is a little off. Maybe a semicolon or em dash?

**Review by charliermarsh — APPROVED** [review]

File: docs/guides/install-python.md (hunk 2 above)
Comment (zanieb) [reply docs/guides/install-python.md:69]:
```suggestion
uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior: uv will use the system Python if it satisfies the requirements of the command invocation. See the [Python discovery](../python-versions.md#discovery-order) documentation for details.
```

File: docs/guides/install-python.md (hunk 1 above)
Comment (BurntSushi) [inline docs/guides/install-python.md:3]: I might use "will" instead of "should" here. I think you use "will" below.

Comment (zanieb) [reply docs/guides/install-python.md:3]: Thanks!

**Pull request merged**